doc/releases/changelog-0.23.0.md

```diff
@@ -489,6 +497,14 @@
   1
   ```
 
+* The `qml.finite_diff` function has been deleted. Please use `qml.gradients.finite_diff` to compute
+  the gradient of tapes of QNodes. Otherwise, manual implementation is required.
+  [(#2464)](https://github.com/PennyLaneAI/pennylane/pull/2464)
+
+* The `get_unitary_matrix` transform has been removed, users should use
+  `qml.matrix` instead.
+  [(#2457)](https://github.com/PennyLaneAI/pennylane/pull/2457)
```

Comment:
```suggestion
* The `get_unitary_matrix` transform has been removed, please use
  `qml.matrix` instead.
  [(#2457)](https://github.com/PennyLaneAI/pennylane/pull/2457)
```

I don't like having the term 'user' appear in the changelog 😆

Reply:
Makes sense, I guess we are talking to users through the release notes, it feels a bit weird to mention them in third person.

Reply:
Haven't found other instances.

Reply:
> I guess we are talking to users through the release notes, it feels a bit weird to mention them in third person.

💯